pkgs/tools/misc/aescrypt/default.nix

```diff
@@ -9,6 +9,12 @@ stdenv.mkDerivation rec {
     sha256 = "1a1rs7xmbxh355qg3v02rln3gshvy3j6wkx4g9ir72l22mp6zkc7";
   };
 
+  makeFlags = [ "CC=cc" ];
```

Comment:
This shouldn't be necessary if the makefile is patched.

Reply:
Looks like my patch isn't actually applying properly. I wonder why? The log from `nix-build` suggests that it's patching, which I'm running with `nix-build -E 'with (import <nixpkgs> {}); callPackage ~/code/open-source/nix-group/nixpkgs/pkgs/tools/misc/aescrypt/default.nix {}'`.
![image](https://user-images.githubusercontent.com/5614134/40579943-4948b638-60e9-11e8-931d-5fec0c988960.png)

I'll try loading it in nix-shell I suppose...

Reply:
You could also use `substituteInPlace` in postPatch instead of a patchfile, since the context of the diff isn't important.